build.sbt

```diff
@@ -83,6 +83,8 @@ lazy val core = project
       "org.apache.lucene" % "lucene-analyzers-common" % luceneVersion,
       "org.ow2.asm"       % "asm-commons"             % "5.2",
       "org.ow2.asm"       % "asm-util"                % "5.2",
+      // TODO remove the locally built snapshot in favour of a release
+      "org.scalameta" %% "lsp4s" % "0.1.0+5-de927af3-SNAPSHOT",
```

Comment:
There is a v0.1.0 release on Maven http://search.maven.org/#artifactdetails%7Corg.scalameta%7Clsp4s_2.12%7C0.1.0%7Cjar